features/promotion/receiving_discount_based_on_items_total.feature

```diff
@@ -5,35 +5,35 @@ Feature: Receiving discount based on items total
     I want to have promotion applied when my items total is qualified
 
     Background:
-        Given the store is operating on a single "France" channel
+        Given the store operates on a single channel in "France"
         And the store has a product "PHP T-Shirt" priced at "€80.00"
         And there is a promotion "Holiday promotion"
 
-    @todo
+    @ui
     Scenario: Receiving discount when buying items for the required total value
-        Given the promotion gives "€10.00" fixed discount to every cart with items total at least "€80.00"
+        Given the promotion gives "€10.00" fixed discount to every order with item total at least "€80.00"
```

Comment:
*Primo:* As I've written [here](https://github.com/Sylius/Sylius/pull/4257#discussion_r53742662), I try to be consistent with previously implemented steps - if we would put ``cart`` here, we should put it everywhere else.
*Secundo:* I was following [rule type name](https://github.com/Sylius/Sylius/blob/43dc7551317d2c7919a29982ec9da7b401ec489a/src/Sylius/Component/Promotion/Model/RuleInterface.php#L21). Perhaps we should discuss, should it be named ``item total`` or ``items total`` ;)